lib/components/map/point-popup.tsx

```diff
@@ -10,17 +14,20 @@ const PopupTitle = styled.div`
   margin-bottom: 6px;
 `
 
-export default function MapPopup ({
+export default function MapPopup({
   mapPopupLocation,
   onSetLocationFromPopup
-}) {
+}: {
+  mapPopupLocation: { name: string }
+  // TODO: what is this function actually
```

Comment:
This sets the "from" or "to" location when you click anywhere on the map. Comment can be changed to adding types for this method signature.